src/app/authentication/login.component.css

```diff
@@ -0,0 +1,35 @@
+.header-1 {
+	color : #04529D;
+	font-weight: 900;
+	font-size: 200%;
+}
+
+.header-2 {
+	color : #FF2C06;
+	font-weight: 900;
+	font-size: 200%;	
+}
+
+.sub-heading-1 {
+	color : #4A4A49;
+	font-weight: 900;	
+}
+
+.desc-1 {
+	color : #9C9999;
```

Comment:
It was just to distinguish the text content but I just realized that the css is the same, will remove the redundancy.